keyboards/handwired/nanggong/nanggong.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards)

This file still needs to be removed.

Reply:
I'm sorry I thought I had deleted it last night, but apparently I haven't committed the delete last night